File: Dataset/Leetcode/train/7/592.py
 class Solution(object):
    def XXX(self, x):
        """
        :type x: int
        :rtype: int
        """
        max1=2**31-1
        min1=-2**31
        n=0
        if x>=0:
            while x!=0:
                n=n*10+x%10
                x=x/10
            if n<=max1:
                return n
            else:
                return 0
        else:
            x=-x
            while x!=0:
                n=n*10+x%10
                x=x/10
            if -n>min1:
                return -n
            else:
                return 0

File: Dataset/Leetcode/train/62/592.java
class Solution {
    public int XXX(int m, int n) {
        int[][] v = new int[m][n];
        return find(v,m-1,n-1);
    }

    private int find(int[][] v, int m, int n) {
        if(m==0 || n == 0) return 1;
        if(v[m][n] == 0)
            v[m][n] = find(v,m-1,n) + find(v,m,n-1);
        return v[m][n];
    }
}

